FE/src/App.js

```diff
@@ -2,6 +2,21 @@ import React, { useEffect, useState } from "react";
 import Header from "./components/Header";
 import TaskList from "./components/TaskList";
 import taskFetcher from "./fetchers/fetchTasks";
+import { createGlobalStyle } from "styled-components";
+
+const GlobalStyle = createGlobalStyle`
+body {
+    background-color: #13293D;
+    margin: 0;
+  }
+
+  #root { 
+    padding-top: 100px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+`;
```

Comment:
Para além de isto estar mal formatado, este "reset" deveria estar no ficheiro `index.css`

```suggestion
const GlobalStyle = createGlobalStyle`
  body {
    background-color: #13293D;
    margin: 0;
  }

  #root { 
    padding-top: 100px;
    display: flex;
    flex-direction: column;
    align-items: center;
  }
`;
```

De nenhuma forma é boa prática ter `padding-top: 100px;` para ocupar o tamanho do header. Dá uma vista de olhos nisto: https://developer.mozilla.org/en-US/docs/Web/CSS/flex-grow